### PR TITLE
Actually display assistant exec in game

### PR DESCRIPTION
**Pull request by Elmon11**

### CODE REVIEW

**Review by Focusvity — CHANGES_REQUESTED** [review]
While this is still pending, you might as well update to correct the spelling from ConfigEntry and other places that requires correction.

**Comment by Elmon11** [comment]
I'm not sure what you're referring to, what spelling errors?

File: src/main/java/me/totalfreedom/totalfreedommod/rank/RankManager.java
Comment (Wild1145) [inline src/main/java/me/totalfreedom/totalfreedommod/rank/RankManager.java:65]: This line here, the config entry has assistant spelt incorrectly, the suggestion is to refactor it to be correctly spelt.

**Review by Wild1145 — CHANGES_REQUESTED** [review]

**Review by Wild1145 — DISMISSED** [review]
Build failure in fUtil

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project TotalFreedomMod: Compilation failure

[ERROR] /home/travis/build/AtlasMediaGroup/TotalFreedomMod/src/main/java/me/totalfreedom/totalfreedommod/util/FUtil.java:[142,151] cannot find symbol

[ERROR]   symbol:   variable SERVER_ASSTISTANT_EXECUTIVES

[ERROR]   location: class me.totalfreedom.totalfreedommod.config.ConfigEntry

[ERROR] -> [Help 1]

[ERROR] 

[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.

[ERROR] Re-run Maven using the -X switch to enable full debug logging.

[ERROR] 

[ERROR] For more information about the errors and possible solutions, please read the following 
```

**Review by Focusvity — APPROVED** [review]
I'm happy with the changes however the PR is on hold until we get FS-143 merged.

**Pull request merged**
